### ISSUE TITLE
Set up Travis-CI

### ISSUE THREAD

**Issue by Eik-S** [tracemap/tracemap-backend#8]
Set up Travis-CI for test integration to github.

**Comment by Eik-S** [comment]
# Testing
 * Unit Tests
   * For every function
 * Functional Tests
   * For every feature

**Comment by Eik-S** [comment]
#5

**Comment by Eik-S** [comment]
Travis set-up is already done in #4, issue closes after #14 is accepted
